pkg/cache/cache.go

```diff
@@ -60,6 +60,7 @@ func (c *serviceAccountCache) Get(name, namespace string) (role, aud string, use
 	klog.V(5).Infof("Fetching sa %s/%s from cache", namespace, name)
```

Comment:
Do we want a very high (i.e. 9) level log that prints out the service account cache on every read?

Reply:
I have been following this logging convention doc https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/logging.md. Anything >=5 is trace level logging. If I enable logLevel 9, I can see HTTP level logs (Request headers). So, I kept it at 5.